docs/sources/tutorials/alerting-get-started-pt3/index.md

```diff
@@ -275,17 +275,11 @@ To follow the above example, we will create notification policies that route ale
    These nested policies should route alert instances where the region label is either us-west or us-east.
 
    {{< admonition type="note" >}}
-   **Note**: In Grafana, each label within a notification policy must have a unique key. If you attempt to add the same label key (e.g., region) with different values (us-west and us-east), only the last entry will be saved, and the previous one will be discarded. This is because labels are stored as associative arrays (maps), where each key must be unique.
+   **Note**: In Grafana, each label within a notification policy must have a unique key. If you attempt to add the same label key (e.g., region) with different values (us-west and us-east), only the last entry is saved, and the previous one is discarded. This is because labels are stored as associative arrays (maps), where each key must be unique.
```

Comment:
```suggestion
   In Grafana, each label within a notification policy must have a unique key. If you attempt to add the same label key (e.g., region) with different values (us-west and us-east), only the last entry is saved, and the previous one is discarded. This is because labels are stored as associative arrays (maps), where each key must be unique.
```

Reply:
always take out the **Note** because the admonition adds it anyway

Reply:
good catch . thank you!